tests/arp/test_unknown_mac.py

```diff
@@ -42,6 +42,24 @@ def wrapper(self, *args):
     return wrapper
 
 
+@pytest.fixture(autouse=True, scope="module")
+def dut_disable_arp_update(rand_selected_dut):
+    """
+    Fixture to disable arp update before the test and re-enable it afterwards
+
+    Args:
+        rand_selected_dut(AnsibleHost) : dut instance
+    """
+    duthost = rand_selected_dut
+    assert duthost.shell("docker exec -t swss supervisorctl stop arp_update")['stdout_lines'][0] \
```

Comment:
If the `arp_update` stop fails here, LINE#59 will not be executed, we'd better to restart `arp_update` in this case.

Reply:
That is the intention - fail fast if any other test somehow disabled `arp_update`, or if it is disabled unintentionally.

Though if you prefer to continue the test, I am open to change it so re-enable it. Please let me know your decision, thanks.

Reply:
Let's restart it

Reply:
Changed so it will allow `arp_update` to remain off and it will restart when the test is done.